### PR TITLE
Fix 2553 hair loss

### PR DESCRIPTION
**Pull request by ismailsunni**
For #2553

### CODE REVIEW

**Comment by ismailsunni** [comment]
Please check and review @timlinux @samnawi @Charlotte-Morgan

**Comment by timlinux** [comment]
Wee nice job @ismailsunni My hair is gone and will never come back, but I am happy you found it!

File: safe/metadata/property/float_tuple_property.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+from types import NoneType
+
+
+from safe.common.exceptions import MetadataCastError
+from safe.metadata.property import BaseProperty
+
+
+class FloatTupleProperty(BaseProperty):
+    """
+    A property that accepts date input
```
Comment (timlinux) [inline safe/metadata/property/float_tuple_property.py]: probably float instead of date here

**Comment by timlinux** [comment]
Wow reviewing your code it is a really subtle bug - thanks @ismailsunni ! LGTM (see small comment about date vs float).

**Comment by samnawi** [comment]
OK @ismailsunni LGTM

I've tested it in both of my Linux and Windows PC.
Also for QGIS 2.8.6 and 2.12.3.

Thank you.

**Pull request merged**